### PR TITLE
Auction House v0.1

### PR DESCRIPTION
**Pull request by mike-metaplex**

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -13,14 +13,16 @@ let package = Package(
             targets: ["Metaplex"]),
     ],
     dependencies: [
-        .package(url: "https://github.com/metaplex-foundation/Solana.Swift.git", branch: "1.3.0"),
+        .package(url: "https://github.com/metaplex-foundation/metaplex-swift-program-library", branch: "feature/update-auctionhouse")
```
Comment (mike-metaplex) [inline Package.swift:16]: Will set this back to `main` once it's merged.

**Comment by ajamaica** [comment]
Looks  like the lib broke horribly. Thanks for this update.

File: Sources/Metaplex/Modules/AuctionHouse/Models/AuctionHouse.swift [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+import AuctionHouse
+import Foundation
+import Solana
+
+extension Auctionhouse {
+    static func pda(creator: PublicKey, treasuryMint: PublicKey) -> Result<PublicKey, Error> {
+        let seeds = [
+            "auction_house".bytes,
+            creator.bytes,
```
Comment (ajamaica) [inline Sources/Metaplex/Modules/AuctionHouse/Models/AuctionHouse.swift:16]: You got it right away. Awesome

**Review by ajamaica — APPROVED** [review]

**Comment by ajamaica** [comment]
This is awesome. It works and de serialize with beet and solita. I am very very happy

**Pull request merged**